lib/passpartu.rb

```diff
@@ -27,23 +30,44 @@ def self.configure
   end
 
   class Config
+    include Passpartu::Initializer
+
     attr_accessor :policy, :raise_policy_missed_error
-    attr_reader :policy_file
+    attr_reader :policy_file, :use_custom_config_file
 
     def initialize
-      @policy_file = './config/passpartu.yml'
+      @policy_file = DEFAULT_CONFIG_FILE_PATH
+      check_or_create_defaults(@policy_file) unless @use_custom_config_file
       @policy = YAML.load_file(policy_file)
       @raise_policy_missed_error = true
+      @use_custom_config_file = false
     end
 
     def policy_file=(file = nil)
-      @policy_file = file || './config/passpartu.yml'
+      @use_custom_config_file = File.file?(file)
+      @policy_file = file || DEFAULT_CONFIG_FILE_PATH
       @policy = YAML.load_file(policy_file)
     end
+
+    def validate_policy
+      raise wrong_config_error unless policy.is_a?(Hash)
+    end
+
+    private
+
+    def wrong_config_error
+      <<-ERROR
+
+        ************************************************************************************
+        !!! Passpartu is not configured properly. Check configuration file: #{@policy_file}
+        :: Configuration example: https://github.com/coaxsoft/passpartu#real-life-example ::
+        ************************************************************************************
+      ERROR
+    end
   end
 
   configure {}
 end
 
-initializer = './config/initializers/passpartu.rb'
-require initializer if File.exist?(initializer)
+require Passpartu::Initializer::DEFAULT_INITIALIZER_PATH
```

Comment:
Could u require `DEFAULT_INITIALIZER_PATH ` without `Passpartu::Initializer`?

Reply:
it will cause unitialized constant error here